dbt_macros/dune/mark_as_spell.sql

```diff
@@ -6,6 +6,7 @@
         {{ trino_properties(properties) }}
       )
     {%- else -%}
+      {%- set properties = { 'dune.data_explorer.category': 'abstraction', 'dune.vacuum' = '{"enabled":true}' } -%}
```

Comment:
i think it's cleaner to add above to line 3, where we already assign `abstraction` category. this will apply to all models that run in prod, i.e. dbt cloud

Reply:
but in general, i see your point. we only add in expose spell macro:
https://github.com/duneanalytics/spellbook/blob/main/dbt_macros/dune/expose_spells.sql#L14

Reply:
we only need to apply to tables not views. agree that this is not beautiful though 😄

Reply:
in `expose_spells` macro, we apply whether it's table or view. seems like it doesn't harm anything? ultimately, i guess either way will be fine. i'm okay with leaving as-is or matching how `expose_spells` does it

Reply:
Update this to be a bit more readable. 

We want to limit the number of tables we list when doing this so that the job runs faster

Reply:
much more readable, thank you